modules/user_manual/pages/online_collaboration.adoc

```diff
@@ -0,0 +1,42 @@
+= Online Collaboration
+:collabora-online-url: https://www.collaboraoffice.com/collabora-online/
+:libreoffice-url: https://www.libreoffice.org/
+
+== Collabora Online
+
+{collabora-online-url}[Collabora Online] is a powerful {libreoffice-url}[LibreOffice]-based online office that supports all major document, spreadsheet and presentation file formats, and is integrable with ownCloud. 
+
+=== Secure View
+
+Secure View provides the ability to restrict access to documents, in _read-only_ shares, so that users can:
+
+. View a file with or without a custom watermark.
```

Comment:
As above, "Secure View" always includes watermarks and the other properties at once. Print/export is an addition to that.